backend-opal/src/server/watchLists.js

```diff
@@ -32,7 +25,7 @@ router.get('/', function(req, res) {
     res.status(403).send(response);
   } else {
     const response = {
-      watchlists: data
+      watchlists: await WatchListsData.getSymbols(UserId)
```

Comment:
@martinupset just use 
```suggestion
      watchlists: await WatchListsData.getSymbols(1)
```

Reply:
ok